charts/datadog/templates/_helpers.tpl

```diff
@@ -814,20 +814,20 @@ false
 Returns whether Remote Configuration should be enabled in the agent
 */}}
 {{- define "datadog-remoteConfiguration-enabled" -}}
-{{- if or .Values.remoteConfiguration.enabled .Values.datadog.remoteConfiguration.enabled -}}
-true
-{{- else -}}
+{{- if or (not .Values.remoteConfiguration.enabled) (not .Values.datadog.remoteConfiguration.enabled) -}}
```

Comment:
Could we make this more explicit by doing this instead?

```
{{- if and .Values.remoteConfiguration.enabled .Values.datadog.remoteConfiguration.enabled -}}
true
```

Also we have a question about intent - right now the logic is such that if either of the parameters is F, then the result will be F. 

F F -> F
F T -> F
T T -> T

However, in the README it says 

> `remoteConfiguration.enabled... Can be overriden by `datadog.remoteConfiguration.enabled` and `clusterAgent.admissionController.remoteInstrumentation.enabled`

Does this imply that F T should result in T?

Reply:
As Remote Config is now on by default in the agent, we want to provide a parameter that lets us **disable** Remote Config is they want to; and we don't want to change two parameters to disable Remote Config.

This is why the logic has changed to give what you have (False having the priority) - we can't really remove parameters, sadly.


As for the simplification, yep this is better let me fix that